bin/run_tests.sh

```diff
@@ -97,11 +97,15 @@ clean_up
 echo "*** STARTING DELAYED_JOB for $PASSENGER_APP_ENV env ***"
 bin/delayed_job restart $PASSENGER_APP_ENV -n 6 || { echo "FAILED to start DELAYED_JOB" >&2; exit 1; } # WARNING: using "restart" with environment of test is a HACK that will prevent delayed_job from running in development mode, for example
 
-echo "Precompiling assets, yarn and webpacker..."
-export NODE_OPTIONS="--max-old-space-size=4096"
-RAILS_ENV=test NODE_ENV=test bin/bundle exec rake assets:clean
-RAILS_ENV=test NODE_ENV=test yarn install --force --trace
-RAILS_ENV=test NODE_ENV=test bin/bundle exec rake assets:precompile
+# If we're only running Ruby tests, then don't compile JavaScript
+if [[ "$TEST_FILEPATH" == "" ]]; then
```

Comment:
FYI - there are some Ruby integration tests that do require JS assets to be compiled.  Any test that issues a local HTTP request to a non-API endpoints (such as `StudyCreationTest` or `TaxonsControllerTest`) will cause `webpack` to autocompile assets if they are not already present.  This change won't result in an error - just want to make sure you know that static assets are needed for more than `yarn ui-test`.

Reply:
Thanks for the note -- I've added it over in https://github.com/broadinstitute/single_cell_portal_core/pull/1079/commits/555d8e6cdadcbb9171d2b66958826596a54d42fc.